pixano/datasets/builders/folders/base.py

```diff
@@ -115,6 +116,7 @@ def generate_data(
         Returns:
             An iterator over the data following data schema.
         """
+        source_id = None
```

Comment:
This is temporary while we don't manage other sources yet ?

Reply:
In line 151 I check if source is None:
- yes: create a source
- no: do nothing as the source is already created.